### PR TITLE
Users/lesam/colors

### PR DESCRIPTION
**Pull request by samclee**
This PR has three changes:
1) The colors of the toolbar and toolbar icons has been changed to be high-contrast, using a palette of  black, white, and blue, all above a luminosity ratio of above 3:1 with each other.
2) The crosses of `RectSelector.ts` have been replaced with `AlternatingCrosses`, a class derived from `Cross`, but with alternating colors to enhance visibility.
3) The keyboard controls for RectSelector have been changed: instead of controlling the keyboard cross with the UHJK keys, users now hold Shift and use the arrow keys.

color
new_cross

### CODE REVIEW

File: src/canvastools/ts/CanvasTools/Selection/Selectors/RectSelector.ts [hunk 1]
```diff
@@ -385,7 +387,7 @@ export class RectSelector extends Selector {
      * @param key string
      */
     private isKeyboardControlKey(key: string) {
-        return key === "u" || key === "h" || key === "j" || key === "k";
+        return key === "ArrowUp" || key === "ArrowDown" || key === "ArrowLeft" || key === "ArrowRight";
```
Comment (soyfestivo) [inline src/canvastools/ts/CanvasTools/Selection/Selectors/RectSelector.ts]: Nice

File: src/canvastools/ts/CanvasTools/Selection/Component/AlternatingCrossElement.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import { IPoint2D } from "../../Interface/IPoint2D";
+
+/**
+ * The alternating cross element for selectors.
+ */
+export class AlternatingCrossElement extends CrossElement {
+    /**
+     * Horizontal line element.
+     */
+    private hl2: Snap.Element;
```
Comment (soyfestivo) [inline src/canvastools/ts/CanvasTools/Selection/Component/AlternatingCrossElement.ts]: What's the 2 for?

Comment (samclee) [reply src/canvastools/ts/CanvasTools/Selection/Component/AlternatingCrossElement.ts]: The CrossElement class already has members `vl` and `hl`, those are its two dotted lines. To achieve the alternating colors, the AlternatingCrossElement has a second set of dotted lines, `vl2` and `hl2` with the opposite color and opposite dotting pattern. Would `vlalt` and `hlalt` be better names?

Comment (soyfestivo) [reply src/canvastools/ts/CanvasTools/Selection/Component/AlternatingCrossElement.ts]: I see, no I think this is a fine name. Although I would probably say what you just said in this comment in the comment above the variable. I'm sure people looking at this code at a later date would ask the same question as seeing as how there are pretty extensive comments in this repo we should continue that pattern.

File: package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "plato-report": "./node_modules/.bin/es6-plato -r -d ./plato-report lib",
     "serve": "webpack-dev-server --config webpack.config.js --env.mode=prod",
     "start": "webpack --config webpack.config.js --env.mode=prod && copyfiles -f dist/ct.js samples/shared/js/",
-    "watch": "onchange src/canvastools/ts/**/*.ts -- npm run start",
+    "watch": "onchange src/canvastools/ts/**/*.ts src/canvastools/css/canvastools.css -- npm run start",
```
Comment (soyfestivo) [inline package.json]: What about the version bump? You need to bump the version to publish

Comment (samclee) [reply package.json]: Oops, thanks for the reminder

File: src/canvastools/css/canvastools.css [hunk 4]
```diff
@@ -545,11 +559,11 @@ svg:not(:root) .menuLayer {
 }
 
 .iconStyle.switch:hover .iconBGRectStyle{
-    fill: rgba(68, 68, 68, 0.5);
+    fill: rgb(21, 127, 240);
```
Comment (pritamDas9191) [inline src/canvastools/css/canvastools.css]: use hex

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 2.2.9
+* Default colors have been changed.
```
Comment (pritamDas9191) [inline CHANGELOG.md]: more descriptive comment on what exactly has changed

**Comment by pritamDas9191** [comment]
Add ct.js and css files in samples folder too.

**Pull request merged**